newIDE/app/src/MainFrame/EditorContainers/EventsFunctionsExtensionEditorContainer.js

```diff
@@ -56,15 +56,15 @@ export class EventsFunctionsExtensionEditorContainer extends React.Component<Ren
     // reload/regeneration of extensions, as changes can have
     // been made inside before it's closed.
     if (this.props.isActive) {
-      this.props.onLoadEventsFunctionsExtensions();
+      return this.props.onLoadEventsFunctionsExtensions();
```

Comment:
You probably don't want to return here - componentWillUnmount will ignore it and this gives the impression the return value if somewhat used.